dipy/viz/horizon/tab/roi.py

```diff
@@ -50,3 +50,7 @@ def build(self, tab_id, tab_ui):
     @property
     def name(self):
         return self.__name
+
+    @property
+    def tab_type(self):
+        return 'rois_tab'
```

Comment:
same as above. To remove, you do not need it.  you can use `isinstance(cls, clstype)`  or if you love string, you can use:
- `myinstance.__class__.__name__`.
- `type(myinstance).__name__`.